desktop/eventListners.js

```diff
@@ -84,6 +84,20 @@ const subscribeToEventListeners = ({ mainWindow }) => {
   ipcMain.on(ipcConsts.SET_NODE_IP, async (event, request) => {
     netService.setNodeIpAddress({ event, ...request });
   });
+
+  ipcMain.on(ipcConsts.NOTIFICATION_CLICK, () => {
+    mainWindow.show();
+    mainWindow.focus();
+  });
+
+  ipcMain.on(ipcConsts.CAN_NOTIFY, (event) => {
+    try {
```

Comment:
isFocused doesn't throw error thus try catch block and CAN_NOTIFY_FAILURE are redundant

Reply:
ok